src/chains.ts

```diff
@@ -131,7 +131,7 @@ const chains: { [key in ChainId]: Chain } = {
     code: 'LUNA',
     nativeAsset: 'LUNA',
     fees: {
-      unit: 'LUNA'
+      unit: 'UST'
```

Comment:
This is not exactly correct because when we send Luna unit is `Luna` when we send UST unit is `UST` when we send ERC20 unit is `Luna`. With this change fee will be always displayed as `UST`